src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -31,12 +31,14 @@ const Body = BaseForm.extend(
       const sendText = ( this.model.get('primaryMode') === 'sms' )
         ? loc('oie.phone.verify.sms.sendText', 'login')
         : loc('oie.phone.verify.call.sendText', 'login');
+      const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
       const extraCssClasses =
         this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
           'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${sendText}
         <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>${this.model.escape('phoneNumber')}</span>
+        <p>${carrierChargesText}</p>
```

Comment:
Please add a class to `<p>` instead of this. It makes the CSS cleaner. Something like this format is needed `okta-form-subtitle--description` (it's the okta way - we use the modifier syntax from BEM).
It will also help in writing selenium tests.

Reply:
sure

Reply:
for simplicity and this one-off we can keep just `p` tag.. my 2 cents

Reply:
Apologies if I was not clear. I was only hinting at the format `--` and not the exact text for the class. `okta-form-subtitle--description` subtitle itself is like a description and it sounds like an overkill

Reply:
i see. Since it's a static content and I don't we are going to test it, I will keep it as just `p` for now. If the need arise to do selenium test on this, then we can add the class later.